README.md

```diff
@@ -2,23 +2,38 @@
 
 ## Getting started
 
-The project uses a virtual environment to isolate package dependencies. To create the virtual environment and install required packages, run the following from a bash shell terminal:
+The project uses poetry to manage package dependencies.
+This can be setup using:
+``` bash
+     $ run poetry install
+```
 
-### On macOS and Linux
-```bash
-$ source setup.sh
+This project uses flask env variables for the Trello API
+These need to be added to a .env file and include:
 ```
-### On Windows (Using Git Bash)
-```bash
-$ source setup.sh --windows
+    * Key
+    * TOKEN
+    * BoardID
+    * ToDoId (List Id)
+    * DoingId (List Id)
+    * DoneId (List Id)
 ```
 
-Once the setup script has completed and all packages have been installed, start the Flask app by running:
-```bash
-$ flask run
```

Comment:
Your README now implies the reader *has to* use Docker. But they can still run the app locally with `poetry run flask run`.